### DISCUSSION TITLE
WARNING: Using multiheads finetuning with a foundation model that is not a Materials Project model

### DISCUSSION THREAD

**Discussion by liubaoshuai1402** [ACEsuit/mace#726] [Q&A]
Dear developers,

  I try to use multiheads finetuning with a foundation model, mace_agnesi_medium.model or mace_agnesi_small.model.
  But MACE 0.39 seems don't recognize them as a MP model and returns a WARNING in the log: Using multiheads finetuning with a foundation model that is not a Materials Project model, need to provied a path to a pretraining file with --pt_train_file.
  Here is my input YAML, I wonder whether I have neglected some important parameters to open multiheads finetuning?
[input.txt](https://github.com/user-attachments/files/17976710/input.txt)
  Any advice will be appreciated, thanks!

**Comment by Ssbban** [comment]
Hi,
I am facing the same issue. Could you please give me some advice how you resolved it? Thanks for your help.

**Reply by ilyes319** [reply]
you need to add --pt_train_file="mp" to your input.

**Reply by Ssbban** [reply]
Hi Ilyes, thanks for your response. I found that the example on the mace user guide web did not use pt_train_file while automatically identified it as a Material Project model. What causes this difference?
```shell
mace_run_train \
    --name="MACE" \
    --foundation_model="mace_agnesi_small.model" \
    --multiheads_finetuning=True \
    --train_file="train.xyz" \
    --valid_fraction=0.05 \
    --test_file="test.xyz" \
    --energy_weight=1.0 \
    --forces_weight=1.0 \
    --E0s="{"1": 130.0 }" \
    --lr=0.01 \
    --scaling="rms_forces_scaling" \
    --batch_size=2 \
    --max_num_epochs=6 \
    --ema \
    --ema_decay=0.99 \
    --amsgrad \
    --default_dtype="float64" \
    --device=cuda \
    --seed=3
```

**Reply by ilyes319** [reply]
new version i did not update the doc yet.

**Reply by Ssbban** [reply]
Hi Ilyes, thank you for the clarification, but I am currently using version 0.3.8 of MACE. Could this version require any additional adjustments to the input settings?

**Reply by liubaoshuai1402** [reply]
> Hi Ilyes, thank you for the clarification, but I am currently using version 0.3.8 of MACE. Could this version require any additional adjustments to the input settings?

mace_run_train -h will output all the related input key words and its meaning

**Reply by Ssbban** [reply]
Thanks
